src/Components/login/Login.js

```diff
@@ -1,62 +1,56 @@
-import React, { useState, Fragment } from "react";
+import React, { useState, Fragment, useEffect } from "react";
 import { Navigate } from "react-router-dom";
 
 import { Button, TextField, InputAdornment } from "@mui/material";
 import VisibilityIcon from "@mui/icons-material/Visibility";
 import { VisibilityOff } from "@mui/icons-material";
 
 import Image from "../image/ImageContainer";
-import { styles } from "../../utils";
+
+import {
+  messageSignInStyle,
+  failedLoginMessageStyle,
+  UserDoesNotExistsMessageStyle,
+} from "./materialStyles";
+
+import { styles, getFromLocalStorage } from "../../utils";
 import "./styleLogin.scss";
 
 const Login = () => {
-  const [messageSignIn, setMessageSignIn] = useState({
-    text: "Enter your detalls below.",
-    messageColor: { color: "rgb(150, 157, 166)" },
-    borderBottonStyle: { borderBottom: "1px solid rgb(179, 175, 177)" },
-  });
+  const [messageSignIn, setMessageSignIn] = useState(messageSignInStyle);
   const [emailText, setEmailText] = useState("");
   const [passwordText, setPasswordText] = useState("");
   const [showPassword, setShowPassword] = useState(false);
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [goCreateAccount, setGoCreateAccount] = useState(false);
-  const { classes } = styles();
 
-  const failedLoginMessage = () => {
-    setMessageSignIn({
-      text: "Oops! That email and password combination is not valid.",
-      messageColor: { color: "rgb(237, 85, 151)" },
-      borderBottonStyle: { borderBottom: "1px solid rgb(237, 85, 151)" },
-    });
-  };
+  const { classes } = styles();
 
-  const UserDoesNotExistsMessage = () => {
-    setMessageSignIn({
-      text: "Oops! Username does not exist",
-      messageColor: { color: "rgb(237, 85, 151)" },
-      borderBottonStyle: { borderBottom: "1px solid rgb(237, 85, 151)" },
-    });
-  };
+  useEffect(() => {
+    const informationUser = getFromLocalStorage();
+    if (informationUser && informationUser.isLoggedIn) {
+      setIsLoggedIn(true);
+    }
+  }, []);
 
   const processLogin = () => {
-    const informationUser = JSON.parse(localStorage.getItem("userInformation"));
-
-    if (informationUser === null) {
-      UserDoesNotExistsMessage();
+    const informationUser = getFromLocalStorage();
+    if (!informationUser) {
+      setMessageSignIn(UserDoesNotExistsMessageStyle);
       return;
     } else {
```

Comment:
you dont need this else since you already have a return on line 40

Reply:
ok